packages/icons/icons.json

```
@@ -4184,5 +4184,20 @@
         "tags": "footnote, notebook, note, reference, text, citation",
         "group": "editor",
         "codepoint": 62294
+    },
+    {
+        "displayName": "Binary",
+        "iconName": "binary",
```

I think the names should be more specific here, it's not immediately obvious that we're dealing with numbers. Unlike the `"floating-point"` icon, in this case there is potential ambiguity when dealing with things like binary trees, etc.

```suggestion
        "displayName": "Binary number",
        "iconName": "binary-number",
```